pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -225,13 +239,54 @@ def counting_pokemon(self, response_dict, id_list):
                     pass
                 else:
                     pokemon = item['inventory_item_data']['pokemon_data']
-                    if pokemon.get('is_egg', False):
-                        continue
-                    id_list.append(pokemon['id'])
+                    callback(pokemon)
 
-        return id_list
 
-    def should_release_pokemon(self, pokemon_name, cp, iv, response_dict):
+    # Returns True if pokemon is better than other_pokemon
+    def _is_better(self, pokemon, other_pokemon):
```

Comment:
A Pokemon isn't better than other pokemon just because it has higher CP. This function name isn't intuitive.